web.py

```diff
@@ -11,8 +11,8 @@
 @app.route('/')
 def home():
     if simulation is None:
-        return render_template('index.html', max_sim=0, num_graphs=0)
-    return render_template('index.html', max_sim=ITERATION_COUNT, num_graphs=len(simulation))
+        return render_template('index.html', max_sim=0, num_graphs=0, last_index=0)
+    return render_template('index.html', max_sim=ITERATION_COUNT, num_graphs=len(simulation), last_index=len(simulation))
```

Comment:
Can you just use `num_graphs` to determine the last index?

Reply:
Yes... Don't know what I was think... It legit the same values next to each other...